client/exercises/links-and-buttons/card-component/card-component--solution.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Card component</title>
+</head>
+
+<body>
+  <article>
+    <time>15 days ago</time>
+    <h2>
+        <a href="#">New Year, New Beginnings:
```

Comment:
Real links for the solution, especially here where we have the original content from Smashing Magazine.